docs/plugins/ingest.asciidoc

```diff
@@ -101,9 +87,7 @@ Throws error when the field is not an array.
 --------------------------------------------------
 {
   "join": {
-    "fields": {
-      "joined_array_field": "other_array_field"
-    }
+    "joined_array_field": "-"
```

Comment:
@martijnvg haha, everything is a key-value and I have two examples. which one do you prefer? the existing solution? 


or this example?
```
"join" : {
  "field" : "joined_array_field",
  "separator" : "-"
}
```